package.json

```diff
@@ -10,12 +10,20 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "@emotion/react": "^11.11.4",
+    "@emotion/styled": "^11.11.5",
     "@fortawesome/fontawesome-free": "^6.5.2",
```

Comment:
We should be deleting any unused packages in our application, we do have a material icons from MUI. We need to update any font awesome icon with the corresponding one from MUI.
When the application grows, having many unneeded packages will increase the overall bundle size of our application.